src/screens/SupportUsScreen/SupportUsSectionHeader.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from "react";
+import { ImageBackground, StyleSheet } from "react-native";
+import type { ImageRef } from "../../data/image-ref";
+import Text from "../../components/Text";
+import { lightNavyBlueColor, whiteColor } from "../../constants/colors";
+
+type Props = {
+  image: ImageRef,
+  title: string
+};
+
+const SupportUsSectionHeader = ({ image, title }: Props) => (
+  <ImageBackground style={styles.image} source={image} resizeMode="cover">
+    <Text type="h1" style={styles.title}>
+      {title}
+    </Text>
+  </ImageBackground>
+);
+
+const styles = StyleSheet.create({
+  image: {
+    aspectRatio: 2 / 1,
+    justifyContent: "flex-end",
+    alignItems: "flex-start"
+  },
+  title: {
+    backgroundColor: whiteColor,
+    color: lightNavyBlueColor,
+    margin: 16,
+    paddingHorizontal: 8,
+    lineHeight: 40
```

Comment:
On iOS this looks a bit top heavy. I suggest using:

```
paddingTop: 4,
lineHeight: 36
```

Existing (with descender text):
<img width="310" alt="40" src="https://user-images.githubusercontent.com/1566539/38799026-326a7936-415b-11e8-8449-1fed1aeee336.png">

With change (with descender text):
<img width="310" alt="36-4" src="https://user-images.githubusercontent.com/1566539/38799048-3e20447c-415b-11e8-8e8e-2e8e0296cc2d.png">

Reply:
Nice. Will try this. Text alignment seems to differ a bit between Android and iOS. So I will need to find a good middle gound, I think.